js/event_target.ts

```diff
@@ -201,8 +201,6 @@ export class EventTarget implements domTypes.EventTarget {
       );
     }
 
-    event.isTrusted = false;
```

Comment:
Hmmm... how does the value get initialised then?  Is it supposed to be `undefined` initially, or `false`?

Reply:
`false`

Reply:
Well isn't it `undefined` though?  Where does it get set to `false`?

Reply:
I add a comment, the default value of `isTrusted` is `false`.

Reply:
in constructor, use `defineProperty` to define on each instance, NOT on the prototype.
https://github.com/denoland/deno/blob/master/js/event.ts#L44